app/components/modules/Header.jsx

```diff
@@ -113,25 +113,27 @@ class Header extends React.Component {
             page_title = `Stolen Account Recovery`;
         } else if (route.page === 'UserProfile') {
             user_name = route.params[0].slice(1);
-            page_title = user_name;
+            const meta_data = this.props.account_meta.getIn([user_name, 'json_metadata']);
+            const meta_name = meta_data ? JSON.parse(meta_data).profile.name : "";
```

Comment:
We need to use `normalizeProfile` to access profile metadata since it's untrusted